ggml/src/ggml-vulkan/vulkan-shaders/dequant_funcs_cm2.comp

```diff
@@ -0,0 +1,305 @@
+
+#include "types.comp"
+
+layout(buffer_reference, std430, buffer_reference_align = 2) buffer decodeBufQ4_0 {
```

Comment:
Is buffer_reference supported without any device extension or minimum Vulkan API version requirement?

Reply:
There's a feature enable for it in VkPhysicalDeviceVulkan12Features, so it's getting filled out when it's queried from the driver.

Reply:
I see, might be good to check for it. It seems most drivers support it at this point, except some phones and SoCs.

Reply:
will do